src/lib.rs

```diff
@@ -230,11 +226,23 @@ impl Mnemonic {
 	/// interpreted in multiple languages, an [Error::AmbiguousWordList] is
 	/// returned, containing the possible languages.
 	pub fn language_of(s: &str) -> Result<Language, Error> {
-		// First we try wordlists that have guaranteed unique words.
 		let first_word = s.split_whitespace().next().unwrap();
 		if first_word.len() == 0 {
 			return Err(Error::BadWordCount(0));
 		}
+
+		// For efficiency reasons, we add a special case for when there's
+		// only a single language enabled.
+		if Language::all().len() == 1 {
```

Comment:
I'm not sure if that's really necessary, but am ok with it (adds a little bit of redundancy).